test/unit/say.result

```diff
@@ -1,67 +0,0 @@
-1..33
-# type: file
-# next: 
```

Comment:
We miss those checks after converting of the test. Checking of those `type`, `next`, `identity`, `facility` tokens seems to be the main thing that is verified by the test. Let's drop this commit from the patchset or convert `note()` calls to 'assertions' (`ok()`, `is()` and so on).

Reply:
Thanks for the catch! I'll drop this commit - it needs some more lave later.